projects/packages/videopress/src/client/admin/components/video-row/index.tsx

```diff
@@ -0,0 +1,263 @@
+		</div>
+	);
+};
+
+const QuickActions = ( { button }: { button: React.ReactNode } ) => {
+	// Hiding it based on Design request:
+	// https://github.com/Automattic/jetpack/issues/25742#issuecomment-1223123815
+	const HIDE_QUICK_ACTIONS = false;
+
+	if ( HIDE_QUICK_ACTIONS ) {
+		return null;
+	}
+
+	return (
+		<div className={ styles.actions }>
+			{ button }
+			<ActionItem icon={ image }>{ __( 'Update thumbnail', 'jetpack-videopress-pkg' ) }</ActionItem>
+			<ActionItem icon={ privacy }>{ __( 'Update privacy', 'jetpack-videopress-pkg' ) }</ActionItem>
+			<ActionItem icon={ trash } className={ styles.trash }>
+				{ __( 'Delete video', 'jetpack-videopress-pkg' ) }
+			</ActionItem>
+		</div>
+	);
+};
+
+const Stats = ( {
+	duration,
+	uploadDate,
+	plays,
+	isPrivate,
+}: {
+	duration: string;
+	uploadDate: string;
+	plays: number;
+	isPrivate: boolean;
+} ) => {
+	const [ isSmall ] = useBreakpointMatch( 'sm' );
+	const durationLabel = __( 'Duration', 'jetpack-videopress-pkg' );
+	const playsLabel = __( 'Plays', 'jetpack-videopress-pkg' );
+	const privacyLabel = __( 'Privacy', 'jetpack-videopress-pkg' );
+
+	return (
+		<div className={ classNames( styles.stats, { [ styles.small ]: isSmall } ) }>
+			<div className={ styles.privacy } aria-disabled={ isSmall ? 'false' : 'true' }>
+				{ isPrivate && ! isSmall && <Icon icon={ privacy } /> }
+				{ isSmall && (
+					<>
+						<span>{ privacyLabel }</span>
+						<span>
+							{ isPrivate
+								? __( 'Private', 'jetpack-videopress-pkg' )
+								: __( 'Public', 'jetpack-videopress-pkg' ) }
+						</span>
+					</>
+				) }
+			</div>
+			<div className={ styles.duration }>
+				<span>{ isSmall && durationLabel }</span>
+				<span>{ duration }</span>
+			</div>
+			{ Number.isFinite( plays ) && (
+				<div className={ styles.plays }>
+					<span>{ isSmall && playsLabel }</span>
+					<span>{ plays }</span>
+				</div>
+			) }
+			{ ! isSmall && <div className={ styles.upload }>{ uploadDate }</div> }
+		</div>
+	);
+};
+
+const VideoRow = ( {
+	videoTitle,
+	posterImage,
+	duration,
+	uploadDate,
+	plays = null,
+	isPrivate = false,
+	onClickEdit,
+}: {
+	videoTitle: string;
+	posterImage: string;
+	duration: number;
+	uploadDate: string;
+	plays: number;
+	isPrivate: boolean;
+	onClickEdit?: () => void;
+} ) => {
+	const [ isSmall ] = useBreakpointMatch( 'sm' );
+	const [ showActions, setShowActions ] = useState( false );
+	const [ keyPressed, setKeyDown ] = useState( false );
+	const [ expanded, setExpanded ] = useState( false );
+	const textRef = useRef( null );
+
+	const durationInMinutesAndSeconds = millisecondsToMinutesAndSeconds( duration );
+	const uploadDateFormatted = dateI18n( 'M j, Y', uploadDate, null );
+	const isSpaceOrEnter = code => code === 'Space' || code === 'Enter';
+	const isEllipsisActive = textRef?.current?.offsetWidth < textRef?.current?.scrollWidth;
+	const showTitleLabel = ! isSmall && isEllipsisActive;
```

Comment:
There seems to be no way to see the full title on mobile when the text is cropped. Maybe only `isEllipsisActive` should be enough?

Reply:
Over doesn’t looks well at mobile, that’s the reason I deactivated it, we should reach out back to design for this one.